hpke.go

```diff
@@ -184,21 +184,38 @@ func keySchedule(suite CipherSuite, mode Mode, sharedSecret, info, psk, pskID []
 	return params, nil
 }
 
+// Context represents an HPKE context encoded on the wire.
+type Context struct {
+	Role                 Role
+	KemId, KdfId, AeadId uint16
```

Comment:
Couple of minor things here:
* I would prefer these on separate lines.
* Go idiom is to capitalize acronyms, so `KEMID`, etc.
* We have types for these (`type KEMID uint16`, etc.), so we should use them.

Reply:
So `kdfId` for normal variable and `KDFID` for exported field structure?

Reply:
Done.

Reply:
`kdfID` for a variable / private field, `KDFID` for exported type/field